player/types.go

```diff
@@ -9,3 +11,46 @@ type PartInfo struct {
 	Player string
 	Part   int64
 }
+
+type PlayerRound struct {
+	ID        int64
+	Status    int64
+	CreatedAt time.Time
+	UpdatedAt time.Time
+}
+
+type PlayerRoundStatus int
+
+func (prs PlayerRoundStatus) Enum() int {
+	return int(prs)
+}
+
+func (aus PlayerRoundStatus) ReflexType() int {
+	return attachmentUploadReflex[aus].ReflexType()
+}
+
+func (PlayerRoundStatus) ShiftStatus() {
```

Comment:
You can combine the statuses so we only have one enum

Reply:
done